src/net.h

```diff
@@ -1054,6 +1084,7 @@ class CConnman
 
     size_t SocketSendData(CNode& node) const EXCLUSIVE_LOCKS_REQUIRED(node.cs_vSend);
     void DumpAddresses();
+    AddedNodeParams GetAddedNodeParams(std::string addedNode);
```

Comment:
```suggestion
    AddedNodeParams GetAddedNodeParams(std::string addedNode) const;
```

Reply:
This became obsolete, since GetAddedNodeParams was removed.